src/components/ModalConfirmation.js

```diff
@@ -0,0 +1,35 @@
+import Modal from "./Modal.js";
+
+export default class ModalConfirmation extends Modal {
+  constructor(modalSelector) {
+    super({ modalSelector });
+    this._modalForm = this._modalElement.querySelector(
+      ".modal__container-form"
+    );
+    this._saveButton = this._modalForm.querySelector(
+      ".modal__container-form-button"
+    );
+  }
+
+  setSubmitAction(action) {
+    this._handleConfirm = action;
+  }
+
+  renderLoading(isLoading) {
```

Comment:
You've implemented the code for rendering loading via the class methods. Great!

Reply:
Thanks!